### DISCUSSION TITLE
BPF per Interface

### DISCUSSION THREAD

**Discussion by argwfm** [Security-Onion-Solutions/securityonion#7415] [Unsupported Versions] [answered]
Is there any way to apply BPF granularly to a bond0 monitoring sub-interface? eth1 and eth2 are monitored via bond0 and looking to drop port 443 on eth2 (external tap) while retaining on eth1 (decrypt). Unable to come up w/ a solution as from what I can tell BPF is applied to bond0 and not the individual monitoring sub-interfaces. Any ideas appreciated!

**Comment by mnasec** [comment, marked as answer]
From my point of view this is a architectural problem. BPF are applied to a process, e.g. to suricata and not to an interface. And suricata is monitoring the bond0 interface. So i think one way is to specify the BPF in the way to catch the different ip addresses you want to be excluded. But if the IP addresses or ports are the same on both interfaces it will not work.
An other possibility may be to use a different forwarding node and apply the BPF filter on minion level and not at the global level.
Hope this helps!
